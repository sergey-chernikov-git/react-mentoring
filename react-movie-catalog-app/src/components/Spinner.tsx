import React from 'react';

export const Spinner = () => {
  return <div className="loader">Loading...</div>;
};
