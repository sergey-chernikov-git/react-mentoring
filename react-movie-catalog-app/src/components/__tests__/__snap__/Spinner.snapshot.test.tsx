import React from 'react';
import { Spinner } from '../../Spinner';
import { render, cleanup } from '@testing-library/react';
import { expect } from '@jest/globals';

afterEach(cleanup);

describe('Spinner Snapshot Test Suit', () => {
  test('renders correctly', () => {
    const { asFragment } = render(<Spinner />);
    expect(asFragment()).toMatchSnapshot();
  });
});
