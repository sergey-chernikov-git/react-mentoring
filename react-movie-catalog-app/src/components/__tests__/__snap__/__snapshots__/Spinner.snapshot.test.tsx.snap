// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Snapshot Test Suit renders correctly 1`] = `
<DocumentFragment>
  <div
    class="loader"
  >
    Loading...
  </div>
</DocumentFragment>
`;
